### PR TITLE
Specify a reviewer and entry_level assignment policy

### PR DESCRIPTION
**Pull request by joshblum**

### CODE REVIEW

File: orchestra/utils/task_lifecycle.py [hunk 1]
```diff
@@ -795,7 +795,10 @@ def submit_task(task_id, task_data, iteration_status, worker):
     task.status = next_status
     task.save()
 
-    if task.status == Task.Status.REVIEWING:
+    if task.status == Task.Status.PENDING_REVIEW:
+        # Check the assignment policy to try to assign a reviewer automatically
+        task = _preassign_workers(task, policy_type='reviewer')
```
Comment (marcua) [inline orchestra/utils/task_lifecycle.py]: it'd be nice to externalize these string somewhere, like POLICY_TYPE_REVIEWER or something

File: orchestra/utils/task_lifecycle.py [hunk 2]
```diff
@@ -896,14 +899,17 @@ def end_project(project_id):
         notify_status_change(task, assignment_history(task))
 
 
-def _preassign_workers(task):
+def _preassign_workers(task, policy_type='entry_level'):
```
Comment (marcua) [inline orchestra/utils/task_lifecycle.py]: i think we should make this arg explicit

File: orchestra/utils/task_lifecycle.py (hunk 1 above)
Comment (joshblum) [inline orchestra/utils/task_lifecycle.py]: I had it in an enum but then none of the other json keys were so i took it out

File: orchestra/utils/task_lifecycle.py (hunk 2 above)
Comment (joshblum) [inline orchestra/utils/task_lifecycle.py]: ok let's do it and i'll add the enum back

**Pull request merged**